README.md

```diff
@@ -40,7 +40,7 @@ __PocketMine-MP is a free, open-source software that creates Minecraft: Pocket E
 [#pmmp + #pocketmine channel WebIRC](http://webchat.freenode.net/?channels=pmmp,pocketmine)
 
 ### Want to contribute?
-* Check the [Contributing Guidelines](CONTRIBUTING.md)
+* Check the [Contributing Guidelines](.github/CONTRIBUTING.md)
```

Comment:
I don't think moving CONTRIBUTING.md is necessary since it has existed here for ages.